lte/gateway/c/oai/tasks/s1ap/s1ap_mme_nas_procedures.c

```diff
@@ -731,167 +851,204 @@ void s1ap_handle_conn_est_cnf(
    */
   //     s1ap_timer_insert(ue_ref->mme_ue_s1ap_id,
```

Comment:
Is this comment relevant here?

Reply:
Used this log when gdb was not working, now removed.